xml/System.Net.Security/SslStream.xml

```diff
@@ -3453,9 +3453,17 @@ Authentication has not occurred.</exception>
         <ReturnType>System.Net.Security.TlsCipherSuite</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the cipher suite resolved to use by this <see cref="T:System.Net.Security.SslStream"/>.</summary>
+        <value>The cipher suite resolved to use by this <see cref="T:System.Net.Security.SslStream"/>.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks
+This property gets the cipher suite that is going to be used in the communication between a client and server by the underlying SSL/TLS implementation. This property is only available after a successful call to <xref:System.Net.Security.SslStream.AuthenticateAsServer%2A> or <xref:System.Net.Security.SslStream.AuthenticateAsClient%2A>.
+
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.InvalidOperationException">The property was accessed before the completion of the authentication process or the authentication process failed.</exception>
```

Comment:
Try to match description with this https://docs.microsoft.com/en-us/dotnet/api/system.net.security.sslstream.keyexchangealgorithm?view=netframework-4.8 which is related property (KeyExchange is implied by the negotiated cipher suite) as much as possible at least

Reply:
I was trying to match against [`HashAlgorithm`](https://docs.microsoft.com/en-us/dotnet/api/system.net.security.sslstream.hashalgorithm?view=netcore-3.0) which is also part of the negotiation.

Reply:
that works too